doc/run/repl.md

```diff
@@ -1,16 +1,18 @@
-# Run SQLFlow REPL
+# Run SQLFlow command-line tool
 
-In addition to building SQLFlow into a gRPC server, accessed via Jupyter Notebook, we could also build it into a command line program, whose `main` function reads SQL statements from the console, evaluates them by calling SQLFlow, and prints the results.  This command-line program makes it easy to debug and profile locally without starting the SQLFlow server and the Jupyter server.  We call this command-line program the SQLFlow REPL. SQLFlow REPL supports automatic code completion to ease applying the underlying powerful AI toolset of SQLFlow.
+We have a command-line tool called `sqlflow` which can access SQLFlow server like we do in Jupyter Notebook.  `sqlflow` works in the well known `REPL` mode, that is to say, reads SQL statements from the console, evaluates them by calling SQLFlow server, and prints the results.  `sqlflow` supports automatic code completion to ease applying the underlying powerful AI toolset of SQLFlow.
```

Comment:
In addition to the Jupyter Notebook magic command, we provide a command-line client `sqlflow` to connect and operate the SQLFlow gRPC server.  Similar to other command-line client tools like `mysql`, `sqflow` implements several interaction modes.

1. `sqlflow -e "SELECT ... TO TRAIN ..." runs the SQLFlow program provided in the command line.
1. `sqlflow -f a.sql` runs the SQLFlow program in a file.
1. `sqflow` starts the REPL mode for user interaction.

In the REPL mode, `sqlflow` supports automatic code completion and other features.

Reply:
Does this work?  `cat a.sql | sqlflow -f -`

Reply:
Yes